### DISCUSSION TITLE
Frontend with QUIC and port 443 - HTTP/3 question

### DISCUSSION THREAD

**Discussion by ammnt** [AdguardTeam/AdGuardHome#3622] [Q&A] [answered]
Hello,

for now we have:
1. Support port 443 for frontend and DoH;
2. Experimental QUIC or DoQ support;
3. Frontend HTTP/2 support;

Suppose we don't need DoH, and we only want to use DoQ on 443 port. Will the frontend work in this case and what about HTTP/3 support?

Thank you.
Cheers😗

**Comment by ameshkov** [comment, marked as answer]
DoQ and DoH3 are different things so I don't think it's a good idea.

At some point when we support H3, the same port will be used for accepting H3 over UDP.
